clang/lib/DPCT/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(RUNTIME_HEADERS
   ${CMAKE_SOURCE_DIR}/../clang/runtime/dpct-rt/include/dpct/sparse_utils.hpp
   ${CMAKE_SOURCE_DIR}/../clang/runtime/dpct-rt/include/dpct/fft_utils.hpp
   ${CMAKE_SOURCE_DIR}/../clang/runtime/dpct-rt/include/dpct/lapack_utils.hpp
```

Comment:
It looks like you have forgotten the other line in this CMakeLists.txt file in `PROCESS_FILES_OUTPUT`:

`${CMAKE_BINARY_DIR}/tools/clang/include/clang/DPCT/group_utils.hpp.inc`